core/src/mindustry/world/blocks/distribution/Sorter.java

```diff
@@ -143,6 +143,18 @@ public void buildConfiguration(Table table){
             ItemSelection.buildTable(table, content.items(), () -> sortItem, item -> tile.configure(lastItem = item));
         }
 
+        @Override
+        public boolean onConfigureTileTapped(Tilec other){
+            if(this == other){
+                lastItem = null;
+                control.input.frag.config.hideConfig();
+                tile.configure(null);
```

Comment:
This can be more concise:

```suggestion
                tile.configure(lastItem = null);
```

Reply:
Dang. That's convenient